flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```

Comment:
ok but if they are auto generated why do we need them in the repo? is something external needed to generate them or something?